lib/utils/patterns.ts

```diff
@@ -288,6 +288,15 @@ export function printErrorMessage(message: string) {
 	console.error(chalk.red(`\n${messages.getHelp}\n`));
 }
 
+/**
+ * Print a friendly error message and exit the CLI with an error code, BYPASSING
+ * error reporting through Sentry.io's platform (raven.Raven.captureException).
+ * Note that lib/errors.ts provides top-level error handling code to catch any
+ * otherwise uncaught errors, AND to report them through Sentry.io. But many
+ * "expected" errors (say, a JSON parsing error in a file provided by the user)
+ * don't warrant reporting through Sentry.io.  For such mundane errors, catch
+ * them and call this function.
+ */
```

Comment:
These kind of things often are better in separate commits.

Reply:
👍